elasticdl/python/tests/example_test.py

```diff
@@ -124,7 +124,7 @@ def distributed_train_and_evaluate(
         )
 
         if dataset == "imagenet":
-            shards = {create_imagenet_recordio_file(16, feature_shape): 16}
+            shards = {create_imagenet_recordio_file(8, feature_shape): 8}
```

Comment:
Why this change?

Reply:
Because  the default memory limit of docker is 2G. Running ResNet50 with batch size 16 uses nearly 2G memory and sometimes it fails because of OOM.

Reply:
Got it.